packages/context/test/unit/context.ts

```diff
@@ -39,6 +39,40 @@ describe('Context', () => {
       const result = ctx.contains('bar');
       expect(result).to.be.false();
     });
+
+    it('returns false when the key is only in the parent context', () => {
+      ctx.bind('foo');
+      const childCtx = new Context(ctx);
+      const result = childCtx.contains('foo');
+      expect(result).to.be.false();
+    });
+  });
+
+  describe('isBound', () => {
+    it('returns true when the key is bound in the context', () => {
+      ctx.bind('foo');
```

Comment:
What happens when you try to inject a key that is bound (but to no value) like here?

Reply:
To be consistent with other methods such as `contains` and `getBinding`, we only test if the key exists.

Reply:
I tried searching the code base for the answer, but alas, I don't see what it could be set to in this case! Interested to find out what though.

Reply:
> I tried searching the code base for the answer, but alas, I don't see what it could be set to in this case! Interested to find out what though.

When a binding does not have any value configured, its `getValue` method throws an error - see here: https://github.com/strongloop/loopback-next/blob/39932be95205f97fc8c85853c8d723c6b28186a8/packages/context/src/binding.ts#L233-L235